docs/openhabian-backup.md

```diff
@@ -1,141 +1,148 @@
+Amanda will combine level 0 of some devices with levels 1 or 2 of others, aiming to have the more or less same amount of data to be backed up every run.
+
+See the Amanda [FAQ](https://wiki.zmanda.com/index.php/FAQ:How_do_I_make_Amanda_do_full_backups_on_weekends_and_incrementals_during_the_week%3F) on full backups for more information.
+
+#### Raw Device Backups
+
+Backing up *raw* devices such as `/dev/mmcblk0` (which is the internal SD card reader of a RPi) can only use a level 0 dump.
+So if you include `/dev/mmcblk0` in your disklist, it'll be backed up on *every* run.
+If you don't want that (as it'll likely consume the by far largest part of your backup run time and capacity) then remove it from the disklist.
+You can create a second Amanda configuration to only include that raw device and run it less often (e.g. monthly).
+
+#### Storage for Amanda
+
+Typically, for a backup system to use this methodology, you need the amount of storage to be 2-3 times as large as the amount of data to be backed up.
+The number of tapes and their capacity (both of which are sort of artificially set when you store to a filesystem) determines how long your storage capacity will last until Amanda starts to overwrite old backups.
+By asking you to enter the total size of the storage area, the Amanda installation routine will compute the maximum amount of data that Amanda will store into each tape subdirectory as (storage size) divided by (number of tapes, 15 by default).
+
+#### Amanda Filesystem Permissions
+
+Amanda was built on top of UNIX and makes use of its user and rights system, so it is very useful to familiarize yourself with that.
+As a general good UNIX practice, you shouldn’t use functional users such as `backup` (the OS uses functional users to execute tasks with specific access rights) for administrative functions.
+Use your personal user instead (i.e `openhabian` by default).
+
+Installation tasks including edits of the Amanda config files, require the use the `root` user.
+Any ordinary user (such as your personal one) can execute commands on behalf of root (and with root permission) by prepending `sudo` to the command.
+As yourself, prepend `sudo -u backup` to execute the following commands as the "backup" user.
+
+### Storage Preparation
+
+Now once you have read up on all of this and feel you have understood this stuff, the next step will be to prepare your storage.
+
+::: important
+You need to provide your storage *before* you install Amanda.
+:::
+
+You have to mount the USB stick or disk from your NAS to a directory on your openHABian box.
+If you don't know what this means in UNIX terms, please do some internet searches now to learn more.
+Your mountpoint should be a directory on your Raspberry Pi with the USB device or NAS mounted to.
+
+If you are a Windows user, please note that CIFS shares are not supported and almost certainly will not work.
```

Comment:
Make this more prominent again.
Again, a major reason for the existing wording is that too many people have missed this info and tried CIFS. We don't want to end up back there. The quality of this documentation is also determined by how much it keeps off *ourselves*,
I don't want to repeatedly read and have to answer posts on the forum when people did it the wrong way because they were unwilling or to lazy to read the full docs (which is something that'll always be the case so we must male sure the relevant information is placed first inside a section ).